packages/ckb/src/utils/cell-dep.ts

```diff
@@ -62,7 +63,13 @@ export const fetchTypeIdCellDeps = async (
   let xudtDep = getXudtDep(isMainnet);
   let uniqueDep = getUniqueTypeDep(isMainnet);
 
-  const cellDepsObj = await fetchCellDepsJson();
+  let cellDepsObj;
+  if (vendorCellDeps) {
+    cellDepsObj = vendorCellDeps;
+  } else {
+    cellDepsObj = await fetchCellDepsJson();
+  }
```

Comment:
```suggestion
  const cellDepsObj = vendorCellDeps ?? await fetchCellDepsJson();
```

Reply:
Addressed it in c04c92e819e01283d8260da624b981a2dc645d76,  please take a look.